pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/wheel/FileBackedWheelCache.java

```diff
@@ -51,6 +54,22 @@ public FileBackedWheelCache(File cacheDir, PythonAbiContainer pythonAbiContainer
         return findWheel(library, version, pythonDetails.getVirtualEnvInterpreter());
     }
 
+    /**
+     * Store the wheel file to cache folder.
+     *
+     * @param wheelFile the wheel file to store
+     */
+    @Override
+    public void storeWheelCache(File wheelFile) {
+        if (wheelFile != null) {
+            try {
+                FileUtils.copyFile(wheelFile, new File(cacheDir, wheelFile.getName()));
```

Comment:
I think we can get by using the standard library from Java 7 and better `Files` and `Paths` interface without introducing another dependency.